fpcare/src/app/modules/registration-status/components/request-status/registration-status.component.html

```diff
@@ -52,7 +52,8 @@ <h2 class="border-bottom">Your Information</h2>
           <common-postal-code name="mailPostal"
             [(ngModel)]="applicant.address.postal"
             label="Mailing Postal Code"
-            [disabled]="disablePhn()">
+            [disabled]="disablePhn()"
+            [required]="disablePhn()">
           </common-postal-code>
```

Comment:
[required]="disablePhn()" should be **!**disabledPhn().  The postal code is required when the PHN is not disabled.  Please fix.

Reply:
Good catch, thanks, fixed.